scripts/install-gometalinter.sh

```diff
@@ -0,0 +1,389 @@
+# Taken from: https://github.com/alecthomas/gometalinter/blob/master/scripts/install.sh
```

Comment:
The shebang (line below) needs to be the first line of the file for it to work. Maybe swap these first two lines.

Reply:
Sure, done.